cmd/status.go

```diff
@@ -128,10 +209,16 @@ func init() {
 		"in combination with --name, show the status of every file in the set")
 	statusCmd.Flags().BoolVarP(&statusIncomplete, "incomplete", "i", false,
 		"only show currently incomplete sets")
+	statusCmd.Flags().BoolVarP(&statusComplete, "complete", "c", false,
```

Comment:
Again, this is fine, but mutually exclusive options it might be better to have a single param (--filter?) that takes an argument to determine which type of filter it is.

Reply:
Fair enough.